rust-runtime/aws-smithy-runtime/src/client/retries/strategy/standard.rs

```diff
@@ -152,35 +182,26 @@ impl RetryStrategy for StandardRetryStrategy {
         // Calculate the appropriate backoff time.
         let backoff = match retry_reason {
```

Comment:
Consider breaking this calculation into a separate function to reduce the size of `should_attempt_retry`.

Reply:
I broke it out. Let me know if it makes things clearer